recipes/search_avail_poi_lists.py

```diff
@@ -0,0 +1,7 @@
+from locussdk import get_avail_poi_lists
+
+
+def search_avail_poi_lists(name):
```

Comment:
would recommend adding typehinting for parameters + a doc string so other users will know what this function + arguments mean like:

```python
from pandas import DataFrame

def search_avail_poi_lists(name: str) -> DataFrame:
"""
Retrieve available POI (Point of interest) lists based on the name provided
"""
```